### ISSUE TITLE
loss is high

### ISSUE THREAD

**Issue by JiteshPshah** [johnsmithm/handwritten-tf-1.0#10]
Why is the loss so high in the training? I have resized images to (36,90). it's around 76-88.

**Comment by johnsmithm** [comment]
you need to change the learning rate, check if the images and labels are correct, change the stride parameter. the result from the readme are from multidimentional lstm.
